src/okteto.ts

```diff
@@ -32,7 +32,7 @@ export const state = {
   failed: 'failed',
 };
 
-var isActive = false;
+var isActive = new Map();
```

Comment:
Now that you change this line, let's fix that `var`.
```suggestion
const isActive = new Map();
```